fish-rust/src/compat.c

```diff
@@ -81,3 +82,130 @@ static const bool uvar_file_set_mtime_hack =
 #endif
 #undef UVAR_FILE_SET_MTIME_HACK
 bool UVAR_FILE_SET_MTIME_HACK() { return uvar_file_set_mtime_hack; }
+
+#ifndef RLIMIT_UNKNOWN
+#define RLIMIT_UNKNOWN -1
+#endif
```

Comment:
I think we can define this unconditionally. It shouldn't be defined by the stdlib.